thoth/common/openshift.py

```diff
@@ -1933,8 +2003,40 @@ def schedule_kebechet_run_url(
             labels={"component": "kebechet"},
         )
 
+    def schedule_thamos_workflow(
+        self, check_run_id: int, repo_url: str, commit_sha: str, installation_id: int
+    ) -> str:
+        """Schedule Thamos Advise Workflow for Qeb-Hwt GitHub App.."""
+        if not self.use_argo:
+            return NotImplementedError
+
+        workflow_id = self.generate_id("")
```

Comment:
It might be a good idea to accept `None` in the `generate_id` method to avoid concatenation with empty string. How will workflow be named?

Reply:
```"thamos-advise-check-run-${EVENT_ID}"```

Reply:
Sounds good to me, just that default value to keep sources clean :) #nitpicking